boards/arm/lpcxpresso54114/lpcxpresso54114_m0_defconfig

```diff
@@ -9,6 +9,7 @@ CONFIG_SOC_LPC54114_M0=y
 CONFIG_SOC_SERIES_LPC54XXX=y
 CONFIG_BOARD_LPCXPRESSO54114_M0=y
 CONFIG_CONSOLE=y
+CONFIG_USE_SEGGER_RTT=y
 CONFIG_RTT_CONSOLE=y
```

Comment:
Can remove `CONFIG_RTT_CONSOLE=y`, since it gets selected by `CONFIG_USE_SEGGER_RTT`